### PR TITLE
Browser:  Use polyfill to support Object.assign in IE11

### PR DESCRIPTION
**Pull request by rushenn**
## Description
Use polyfill to support Object.assign in IE11

## Motivation and Context
https://github.com/minio/minio/issues/3898

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: browser/app/index.js [hunk 1]
```diff
@@ -13,44 +13,33 @@
...
-
+import 'babel-polyfill'
 import './less/main.less'
-
 import React from 'react'
 import ReactDOM from 'react-dom'
 import thunkMiddleware from 'redux-thunk'
 import createStore from 'redux/lib/createStore'
 import applyMiddleware from 'redux/lib/applyMiddleware'
-
```
Comment (harshavardhana) [inline browser/app/index.js]: These spaces were added on purpose to help readability, removing them will create a large import blob which would hurt readability. You should revert these changes.

File: browser/.babelrc [hunk 2]
```diff
@@ -5,4 +5,4 @@
     ],
 
     "plugins": ["transform-object-rest-spread"]
-}
+}
```
Comment (harshavardhana) [inline browser/.babelrc]: Unnecessary space character..

**Review by harshavardhana — CHANGES_REQUESTED** [review]
Formatting issues..

**Comment by rushenn** [comment]
@harshavardhana Done the formatting, please check now

**Review by harshavardhana — APPROVED** [review]

**Review by krishnasrinivas — APPROVED** [review]
